src/client/js/partials/exposure-scan.js

```diff
@@ -0,0 +1,109 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/** @type {HTMLDivElement} */
+// @ts-ignore: We guard against a null value by not calling init():
+const exposureScanPartial = document.querySelector("[data-partial='exposureScan']")
+
+if (exposureScanPartial) {
+  init()
+}
+
+async function init () {
+  const urlParams = new URLSearchParams(document.location.search)
+  const emailFromUrl = urlParams.get('email')
+  /** @type {HTMLTemplateElement} */
+  const dataEl = exposureScanPartial.querySelector('#data')
+  dataEl.dataset.email = emailFromUrl
+  const sanitisedEmail = dataEl.dataset.email.trim()
```

Comment:
-1 for Commonwealth spelling (sanitise vs. sanitize :P)

Reply:
We should have a linter / autofix for this :D

Reply:
Oops sorry, but who came up with the idea of having so many variants of what is already my second language? :P 0dabfdd48

(Actually, a spell checker for variable names isn't a [terrible idea](https://en.wikipedia.org/wiki/HTTP_referer#Etymology).)